bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/TreeItem.java

```diff
@@ -1821,14 +1821,6 @@ private static void handleDPIChange(Widget widget, int newZoom, float scalingFac
 	if (!(widget instanceof TreeItem treeItem)) {
 		return;
 	}
-	Image[] images = treeItem.images;
-	if (images != null) {
-		for (Image innerImage : images) {
-			if (innerImage != null) {
-				Image.win32_new(innerImage, newZoom);
-			}
-		}
-	}
```

Comment:
Where will these images be created then?

Reply:
Its not creating the image. it is just calling the handleDpiChange method to set the right handle to the image as per the zoom level, and since the way we obtain the right handle of the image has changed as per the previous explanation, this can be removed.